LuisValgoi/ui5-webcomponents-react-seed
# [Issue-58] Change Colors / Theme Programmatically

## Description

I created a new dialog to switch the application Theme.
The name of the component is `LanguageSwitchDialog` and it pops up when clicking in the Theme Switch button in the user settings dropdown.
Once the Dialog is opened, the user is able to select any of the 5 themes which are supported by the WC React Framework.
`sap_fiori_hcw` and `sap_fiori_hcb` are also available by the framework, but I did not added them since `sap_fiori_3_hcb` and `sap_fiori_3_hcw` already offer a similar theme.
![2020-10-07_17-22-04](https://user-images.githubusercontent.com/16123699/95383514-ada64f00-08c1-11eb-8e88-7ad596740bff.gif)

## Config

Are you adding a configuration file?

- [ ] Have you updated the `publish` scripts?

Are you adding a new feature

- [ ] Have you updated the `README.md`?

## Testing

- Open the Theme Switch Dialog
- Change the Theme and confirm the update

## Automated Tests

- [x] Select this if your code has automated Tests for it.

## References

- [#58 ](https://github.com/LuisValgoi/ui5-webcomponents-react-seed/issues/58)
